action/protocol/account/transfer.go

```diff
@@ -66,6 +66,7 @@ func (p *Protocol) handleTransfer(ctx context.Context, tsf *action.Transfer, sm
 
 	if fCtx.FixGasAndNonceUpdate || tsf.Nonce() != 0 {
 		// update sender Nonce
+		sender.ConvertFreshAccountToZeroNonceType(tsf.Nonce())
```

Comment:
if acc.IsNewbieAccount() && acc.AccountType() == 0 && useZeroNonce {
	acc.ConvertFreshAccountToZeroNonceType()
}
// Make sure the value matches if acc.PendingNonce() is called

Reply:
`acc.IsNewbieAccount() && acc.AccountType() == 0` is duplicate with what's checked inside `ConvertFreshAccountToZeroNonceType`

added the `useZeroNonce` flag check